conformance/README.md

```diff
@@ -50,8 +49,11 @@ update-nkg-manifest            Update the NKG deployment manifest image name and
 | GATEWAY_CLASS           | nginx                                                                                                         | The gateway class that should be used for the tests                                                                       |
 | SUPPORTED_FEATURES      | HTTPRoute,HTTPRouteQueryParamMatching, HTTPRouteMethodMatching,HTTPRoutePortRedirect, HTTPRouteSchemeRedirect | The supported features that should be tested by the conformance tests. Ensure the list is comma separated with no spaces. |
 | EXEMPT_FEATURES         | ReferenceGrant                                                                                                | The features that should not be tested by the conformance tests                                                           |
-| NGINX_IMAGE             | as defined in the ../deploy/manifests/deployment.yaml file                                                    | The NGINX image for the NKG deployments                                                                                   |
-| NKG_DEPLOYMENT_MANIFEST | ../deploy/manifests/deployment.yaml                                                                           | The location of the NKG deployment manifest                                                                               |
+| NGINX_IMAGE             | as defined in the provisioner/static-deployment.yaml file                                                    | The NGINX image for the NKG deployments                                                                                   |
```

Comment:
neat: previous version had all column delimiters aligned. is it possible to retain that?